docs/openapi/open-api.yml

```diff
@@ -69,3 +138,68 @@ components:
         - mSettings
         - mStandings
         - mTeam
+        - players_wl
+        - proTeamSchedules_wl
+        - mDraftDetail
+    
+    TransactionPayload:
+      description: Payload for POST request body to change a lineup
+      type: "object"
+      required:
+        - isLeagueManager
+        - teamId
+        - type
+        - memberId
+        - scoringPeriodId
+        - executionType
+        - items
+      properties:
+        isLeagueManager: 
+          description: not sure if this can be boolean and not sure what happens if its true
+          type: string
+          enum:
+            - "false"
+            - "true" 
+        teamId:
+          description: the team id for the roster to be updated
+          type: integer
+        type: 
+          description: TBD, maybe LM as true can make retroactive changes, make add/drops
+          enum:
+            - "FUTURE_ROSTER" 
+        memberId: 
+          description: SWID
+          type: string
+        scoringPeriodId: 
```

Comment:
Do you have an example value for this? Its unclear if this is some millisecond value the day out of 365 or something else.

Reply:
I will add sample, I think its league day...like 10th day of the fantasy season is 10... '&teamId=5&seasonId=2022&scoringPeriodId=84...' for example